src/zenml/zen_stores/migrations/versions/7cea2e1c2007_add_service_accounts.py

```diff
@@ -0,0 +1,65 @@
+"""add service accounts [7cea2e1c2007].
+
+Revision ID: 7cea2e1c2007
+Revises: 0.45.6
+Create Date: 2023-11-03 19:30:58.523804
+
+"""
+import sqlalchemy as sa
+import sqlmodel
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "7cea2e1c2007"
+down_revision = "0.45.6"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
```

Comment:
Is this complete? I can't find the APIKey table

Reply:
I think you meant to ask about the `ServiceAccount` table ? Service accounts and user accounts share the same `User` table.

Reply:
No this question was about the Table for API Keys, I couldn't find any mention  of it in the migrations. Does that table already exist?

Reply:
Isn't that the only thing that this migration script is doing though, it creates an `api_key` table :P

Reply:
feeling so stupid rn, what the hell was I thinking